### ISSUE TITLE
QWEN2-VL Model Inference

### ISSUE THREAD

**Issue by will-wiki** [vllm-project/vllm#9330]
### Your current environment

<details>
<summary>The output of `python collect_env.py`</summary>

```text
PyTorch version: 2.4.0+cu118
Is debug build: False
CUDA used to build PyTorch: 11.8
ROCM used to build PyTorch: N/A

OS: CentOS Linux 7 (Core) (x86_64)
GCC version: (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)
Clang version: Could not collect
CMake version: version 3.21.4
Libc version: glibc-2.17

Python version: 3.9.13 | packaged by conda-forge | (main, May 27 2022, 16:58:50)  [GCC 10.3.0] (64-bit runtime)
Python platform: Linux-5.10.0-1.0.0.32-x86_64-with-glibc2.17
Is CUDA available: True
CUDA runtime version: 11.8.89
CUDA_MODULE_LOADING set to: LAZY
GPU models and configuration: 
GPU 0: Tesla V100-SXM2-32GB
GPU 1: Tesla V100-SXM2-32GB
GPU 2: Tesla V100-SXM2-32GB
GPU 3: Tesla V100-SXM2-32GB
GPU 4: Tesla V100-SXM2-32GB
GPU 5: Tesla V100-SXM2-32GB
GPU 6: Tesla V100-SXM2-32GB
GPU 7: Tesla V100-SXM2-32GB

Nvidia driver version: 535.129.03
cuDNN version: Probably one of the following:
/usr/lib/libcudnn.so
/usr/lib64/libcudnn.so
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

CPU:
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                96
On-line CPU(s) list:   0-95
Thread(s) per core:    2
Core(s) per socket:    24
Socket(s):             2
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 85
Model name:            Intel(R) Xeon(R) Gold 6271C CPU @ 2.60GHz
Stepping:              7
CPU MHz:               1715.909
CPU max MHz:           3900.0000
CPU min MHz:           1000.0000
BogoMIPS:              5200.00
Virtualization:        VT-x
L1d cache:             32K
L1i cache:             32K
L2 cache:              1024K
L3 cache:              33792K
NUMA node0 CPU(s):     0-95
Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 cdp_l3 invpcid_single intel_ppin ssbd mba ibrs ibpb stibp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm mpx rdt_a avx512f avx512dq rdseed adx smap clflushopt clwb intel_pt avx512cd avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local dtherm ida arat pln pts pku avx512_vnni md_clear flush_l1d arch_capabilities

Versions of relevant libraries:
[pip3] numpy==1.26.3
[pip3] nvidia-cublas-cu11==11.11.3.6
[pip3] nvidia-cuda-cupti-cu11==11.8.87
[pip3] nvidia-cuda-nvrtc-cu11==11.8.89
[pip3] nvidia-cuda-runtime-cu11==11.8.89
[pip3] nvidia-cudnn-cu11==9.1.0.70
[pip3] nvidia-cufft-cu11==10.9.0.58
[pip3] nvidia-curand-cu11==10.3.0.86
[pip3] nvidia-cusolver-cu11==11.4.1.48
[pip3] nvidia-cusparse-cu11==11.7.5.86
[pip3] nvidia-ml-py==12.560.30
[pip3] nvidia-nccl-cu11==2.20.5
[pip3] nvidia-nvtx-cu11==11.8.86
[pip3] pynvml==11.5.0
[pip3] pyzmq==26.2.0
[pip3] torch==2.4.0+cu118
[pip3] torchaudio==2.4.0+cu118
[pip3] torchvision==0.19.0+cu118
[pip3] transformers==4.45.0.dev0
[pip3] triton==3.0.0
[conda] cudatoolkit               11.0.221             h6bb024c_0  
[conda] mkl                       2018.0.0             hb491cac_4  
[conda] mkl-service               1.1.2            py36h17a0993_4  
[conda] numpy                     1.19.5                   pypi_0    pypi
[conda] numpydoc                  0.7.0            py36h18f165f_0  
[conda] pyzmq                     16.0.2           py36h3b0cf96_2  
[conda] torch                     1.6.0+cu101              pypi_0    pypi
[conda] torchvision               0.7.0+cu101              pypi_0    pypi
[conda] transformers              4.6.0                    pypi_0    pypi
ROCM Version: Could not collect
Neuron SDK Version: N/A
vLLM Version: 0.6.1.post2@9ba0817ff1eb514f51cc6de9cb8e16c98d6ee44f
vLLM Build Flags:
CUDA Archs: Not Set; ROCm: Disabled; Neuron: Disabled
GPU Topology:
GPU0    GPU1    GPU2    GPU3    GPU4    GPU5    GPU6    GPU7    NIC0    NIC1    CPU Affinity    NUMA Affinity   GPU NUMA ID
GPU0     X      NV2     NV2     NV1     SYS     NV1     SYS     SYS     PIX     PIX     0-95    0               N/A
GPU1    NV2      X      NV1     NV2     NV1     SYS     SYS     SYS     PIX     PIX     0-95    0               N/A
GPU2    NV2     NV1      X      NV1     SYS     SYS     SYS     NV2     SYS     SYS     0-95    0               N/A
GPU3    NV1     NV2     NV1      X      SYS     SYS     NV2     SYS     SYS     SYS     0-95    0               N/A
GPU4    SYS     NV1     SYS     SYS      X      NV2     NV2     NV1     SYS     SYS     0-95    0               N/A
GPU5    NV1     SYS     SYS     SYS     NV2      X      NV1     NV2     SYS     SYS     0-95    0               N/A
GPU6    SYS     SYS     SYS     NV2     NV2     NV1      X      NV1     SYS     SYS     0-95    0               N/A
GPU7    SYS     SYS     NV2     SYS     NV1     NV2     NV1      X      SYS     SYS     0-95    0               N/A
NIC0    PIX     PIX     SYS     SYS     SYS     SYS     SYS     SYS      X      PIX
NIC1    PIX     PIX     SYS     SYS     SYS     SYS     SYS     SYS     PIX      X 

Legend:

  X    = Self
  SYS  = Connection traversing PCIe as well as the SMP interconnect between NUMA nodes (e.g., QPI/UPI)
  NODE = Connection traversing PCIe as well as the interconnect between PCIe Host Bridges within a NUMA node
  PHB  = Connection traversing PCIe as well as a PCIe Host Bridge (typically the CPU)
  PXB  = Connection traversing multiple PCIe bridges (without traversing the PCIe Host Bridge)
  PIX  = Connection traversing at most a single PCIe bridge
  NV#  = Connection traversing a bonded set of # NVLinks

NIC Legend:

  NIC0: mlx5_0
  NIC1: mlx5_1
```

</details>

### 🐛 Describe the bug

The script to start the service
```
python -m vllm.entrypoints.openai.api_server \
   --served-model-name Qwen2-VL-7B-Instruct \
   --model /data/CodeSpace/models/Qwen2-VL-7B-Instruct \
   --dtype float16 \
   --port ${deploy_port} \
   --gpu-memory-utilization 0.998 \
   --enable-prefix-caching
```

v100 machine, qwen2-vl model, 7b model takes 2.47s per image, 2b model takes 2.3s per image. Is this as expected, why is 2b model only a little faster than 7b model

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
To separate the effects of latency and throughput, I suggest you run the benchmark such as those described in #8495.

**Comment by gongbudaizhe** [comment]
Hi, Did you have this problem when using vLLM Version: 0.6.1.post2

https://github.com/vllm-project/vllm/issues/8281

**Comment by DarkLight1337** [comment]
This should not occur in vLLM v0.6.3 or above anymore.

**Comment by Edwardmark** [comment]
> anymore

could you please explain why is that? @DarkLight1337

**Comment by DarkLight1337** [comment]
The config got patched, see the thread in #8281
